bin/fake-basket-server.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var url = require('url');
```

Comment:
Can you document the need for both a `basket-proxy-server` and a `fake-basket-server`?